FFXIVClientStructs/FFXIV/Client/UI/Misc/ItemFinderModule.cs

```diff
@@ -22,10 +22,12 @@ public unsafe partial struct ItemFinderModule {
     [FieldOffset(0xA5)] public bool ShouldResetInvalid; // only clears data if player does not meet criteria (for example: has glamour dresser NOT unlocked)
     [FieldOffset(0xA6)] public byte UnkA6;
     [FieldOffset(0xA7)] public byte UnkA7;
-    [FieldOffset(0xA8)] public nint Retainer;
-    [FieldOffset(0xB0)] public long RetainerCount;
-    [FieldOffset(0xB8)] public nint RetainerInventory;
-    [FieldOffset(0xC0)] public long RetainerInventoryCount;
+    [FieldOffset(0xA8)] public StdList<ulong> UpdatedRetainerIds;
+    [FieldOffset(0xA8), Obsolete("Use UpdatedRetainerIds instead")] public nint Retainer;
+    [FieldOffset(0xB0), Obsolete("Use UpdatedRetainerIds.LongCount instead")] public long RetainerCount;
```

Comment:
```suggestion
    [FieldOffset(0xB0), Obsolete("Use UpdatedRetainerIds.Head instead")] public long RetainerCount;
```

StdList does not use the same structure as StdMap when it comes to where the count is stored